### PR TITLE
Add option to use custom headers when fetching PDF

### PR DESCRIPTION
**Pull request by mathiasbno**
The title is kind of self explanatory but this adds an option for using custom headers in the request for the PDF.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -24,7 +24,8 @@ <h1 class="h0 mt0 mb0 white inline">AngularJS PDF Viewer</h1>
         delegate-handle="my-pdf-container"
         url="pdfUrl"
         scale="1"
-        show-toolbar="true"></pdf-viewer>
+        show-toolbar="true"
+        headers="{'x-you-know-whats-awesome': 'EVERYTHING'}"></pdf-viewer>
```
Comment (fespinoza) [inline src/index.html]: awesome

**Comment by winkerVSbecks** [comment]
:+1: thanks for the PR

**Comment by mathiasbno** [comment]
Hi @winkerVSbecks, could you release a new version including this fix so we could use bower update to get the code :)

**Comment by winkerVSbecks** [comment]
done

**Comment by mathiasbno** [comment]
Thanks!

The best

**Comment by mathiasbno** [comment]
Ohay @winkerVSbecks, we still have a little trouble getting this feature trough bower. Are you sure you added/compiled this fix to 0.2.3?

**Comment by winkerVSbecks** [comment]
@mathiasbno sorry about that. I forgot to run `dist` before releasing. I'll update shortly.

**Comment by winkerVSbecks** [comment]
@mathiasbno just published the 0.2.4 release with the headers option.

**Comment by mathiasbno** [comment]
Thanks, you rock :)

**Pull request merged**